app/views/catalog/_metadata_block.html.erb

```diff
@@ -6,8 +6,10 @@
     <div class="card-body">
       <dl>
         <% fields.each do |field_name, field| %>
-          <dt class="blacklight-<%= field_name.parameterize %>"><%= render_document_show_field_label document, field: field_name %></dt>
-          <dd class="blacklight-<%= field_name.parameterize %>"><%= doc_presenter.field_value field %></dd>
+          <div class="card-line">
```

Comment:
`card-line` could be changed to be Bootstrap's `row` class. There are certain exception metadata blocks that we've rerouted away from this generic metadata block file (`_access_and_copyright_block.html.erb`, for example), so can you make sure that all blocks have been changed?

Reply:
This has been addressed and in the next commit.